content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md

```diff
@@ -1,197 +1,237 @@
 ---
-title: Lightstep
-description: How to configure the proxies to send tracing requests to Lightstep.
+title: Cloud Observability from ServiceNow
+description: How to configure the proxies to send tracing requests to Cloud Observability (formerly Lightstep).
 weight: 11
-keywords: [telemetry,tracing,lightstep]
+keywords: [telemetry,tracing,lightstep, servicenow, cloud observability]
 aliases:
  - /docs/tasks/telemetry/distributed-tracing/lightstep/
 owner: istio/wg-policies-and-telemetry-maintainers
 test: no
 ---
 
-{{< boilerplate telemetry-tracing-tips >}}
+You use the [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) to send Istio metrics to Cloud Observability using Istio proxies within each pod. Metrics are collected and forwarded to the OpenTelemetry Collector, which acts as a central collection and processing point and then sends those metrics to Cloud Observability.
```

Comment:
The folder content/en/docs/tasks/observability/distributed-tracing/lightstep is for tracing. Consider placing the metric guide into a different folder.

Reply:
Moved to the metrics section

Reply:
Removed references to Lightstep in site where it was mentioned in the context of tracing (this also fixed broken links caused when moving the topics from Tracing to Metrics). There is still a broken link in `/news/releases/1.1.x/announcing-1.1/change-notes/`, but I don't think I should change that, right?